### PR TITLE
Updating TF syncbn for TF2/Keras compatibility. Updating code attribution.

### PR DESCRIPTION
**Pull request by romerojosh**
This PR updates the TF sync BN implementation to derive from `tf.keras.layers.BatchNormalization` for Keras/TF2 compatibility.

Additionally, original code source was from another repository, so updating attribution/copyright to reflect this: https://github.com/google/automl/blob/master/efficientdet/utils.py

### CODE REVIEW

**Comment by DEKHTIARJonathan** [comment]
LGTM
